### DISCUSSION TITLE
Adding StatixOS tm-qpr3 support

### DISCUSSION THREAD

**Discussion by sajidshahriar72543** [ROM-builders/temporary#24089] [General]
https://github.com/ROM-builders/temporary/pull/24068

Please review the changes. I hope it is worth merging.

Thanks :)

**Comment by Apon77** [comment]
non popular roms are not added normally.

**Comment by sajidshahriar72543** [comment]
StatiX is a well reputed rom. Just because of a bit complex to build, average rom builders don't build it.
